node/src/reactor/validator/memory_metrics.rs

```diff
@@ -20,8 +20,8 @@ pub(super) struct MemoryMetrics {
     mem_storage: IntGauge,
     /// Estimated heap memory usage of contract runtime component.
     mem_contract_runtime: IntGauge,
-    /// Estimated heap memory usage of api server component.
-    mem_api_server: IntGauge,
+    /// Estimated heap memory usage of rpc server component.
+    mem_rpc_server: IntGauge,
```

Comment:
We don't want to include the REST or event stream servers in here?

Reply:
Added.